IlmBase/IlmThread/IlmThreadPool.cpp

```diff
@@ -67,17 +67,17 @@ struct TaskGroup::Data
      Data ();
     ~Data ();
     
-    void    addTask () ;
-    void    removeTask ();
 #ifndef ILMBASE_FORCE_CXX03
+    void addTask () ;
+    void removeTask ();
```

Comment:
The `addTask` and `removeTask` functions are now conditionally excluded when compiling with `ILMBASE_FORCE_CXX03`, which causes build failures. This should fix that:

```diff
diff --git a/IlmBase/IlmThread/IlmThreadPool.cpp b/IlmBase/IlmThread/IlmThreadPool.cpp
index 1111111..2222222 100644
--- a/IlmBase/IlmThread/IlmThreadPool.cpp
+++ b/IlmBase/IlmThread/IlmThreadPool.cpp
@@ -67,9 +67,9 @@ struct TaskGroup::Data
      Data ();
     ~Data ();
     
-#ifndef ILMBASE_FORCE_CXX03
     void addTask () ;
     void removeTask ();
+#ifndef ILMBASE_FORCE_CXX03
     std::atomic<int> numPending;
 #else
     int numPending;     // number of pending tasks to still execute
```

(sorry for not catching this earlier, I excluded commit f1180f89477dceaa6554d93b7a63c378b30edb11 when I tested this PR)